compiler/src/main/java/com/squareup/anvil/compiler/codegen/dagger/MembersInjectorGenerator.kt

```diff
@@ -172,6 +192,19 @@ internal class MembersInjectorGenerator : PrivateCodeGenerator() {
                           .addMember("%S", property.requireFqName())
                           .build()
                       )
+                      .apply {
+                        val hasNonInjectAnnotation = property.annotationEntries
+                          .filterNot { it.isInject(module) }
+                          .isNotEmpty()
```

Comment:
This check is a little strange, I've added a `KtAnnotationEntry.isQualifier()` function.